doc/symfony-cache-configuration.rst

```diff
@@ -143,6 +146,79 @@ one of ``client_ips`` or ``client_matcher``*.
 
   **default**: ``PURGE``
 
+
+Purge tags (cache invalidation using tags)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. warning::
+
+    You need at least Symfony 3.4 to use this feature!
+
+
+Symfony's `HttpCache` does not support tags based cache invalidation by default.
+However, this library ships with a `TaggableStore` and a corresponding
+`PurgeTagsListener` which provide this functionality.
+
+Purging tags is only allowed from the same machine by default.
+You can configure the listener just the same as the `PurgeListener` plus the
+`purge_tags_method` and `purge_tags_header`:
+
+* **client_ips**: String with IP or array of IPs that are allowed to
+  purge the cache.
+
+  **default**: ``127.0.0.1``
+
+* **client_matcher**: RequestMatcherInterface that only matches requests that are
+  allowed to purge.
+
+  **default**: ``null``
+
+* **purge_tags_method**: HTTP Method used with purge tags requests.
+
+  **default**: ``PURGETAGS``
+
+* **purge_tags_header**: HTTP Header that contains the comma-separated tags to purge.
+
+  **default**: ``X-Cache-Tags``
+
+For this to work, you have to use the `TaggableStore` as well as the `PurgeListener`
+so your `AppCache` should end up looking like this::
+
+    use FOS\HttpCache\SymfonyCache\TaggableStore();
+    use FOS\HttpCache\SymfonyCache\PurgeTagsListener();
+
+    // ...
+
+    /**
+     * Overwrite constructor to register the TaggableStore.
+     */
+    public function __construct(
+        HttpKernelInterface $kernel,
+        SurrogateInterface $surrogate = null,
+        array $options = []
+    ) {
+        $store = new TaggableStore($kernel->getCacheDir());
+
+        parent::__construct($kernel, $store, $surrogate, $options);
+
+        $this->addSubscriber(new PurgeTagsListener());
+    }
+
+.. note::
+
+    Even if you do not need tags based cache invalidation, you might still be
+    interested in choosing the `TaggableStore` over the default `Store` that
+    Symfony ships with. That's because `TaggableStore` also prunes expired
+    entries on a regular basis which is something the default `Store` does not.
+    The default `Store` keeps filling up your file system without ever cleaning
+    up expired cache entries. The `TaggableStore` counts all the cache write
+    operations (so fetching items from the cache is not slowed down) and after
+    reaching a configurable threshold (default `500`), it prunes expired data.
+    This means that after every `500` HTTP cache writes, your file system directory
+    will be cleaned up and thus kept in good shape.
+    You can configure the prune threshold by providing a different threshold as
+    second argument to the constructor of `TaggableStore`.
```

Comment:
this seems a bit hidden to me. i think the taggable store should have its own documentation section somewhere on this page and the note should just reference that section.

Reply:
Updated.